### ISSUE TITLE
Problem in creating custom scoring function

### ISSUE THREAD

**Issue by Nirvana2211** [EpistasisLab/tpot#921]
[provide general introduction to the issue and why it is relevant to this repository]

I am trying to create a simple custom scoring function and getting the following error:

###
ValueError: 'correlation' is not a valid scoring value. Use sorted(sklearn.metrics.SCORERS.keys()) to get valid options.

###
The scoring function I have created uses the template from spot documentation.

###

def correlation(y_true, y_pred):
    return 1 + np.sum(np.multiply(y_true, y_pred)) / len(y_true)

my_correlation = make_scorer(correlation, greater_is_better=True)

pipeline_optimizer = TPOTRegressor()

pipeline_optimizer = TPOTRegressor(generations=20, population_size=50, cv=5,   random_state=0, verbosity=3, scoring=my_correlation,n_jobs = 30)

Can you please help

**Comment by weixuanfu** [comment]
I think this issue is related to #906 and it was fixed in dev branch. Could you please try dev branch? You may install dev branch via the command below.

```
pip install --upgrade --no-deps --force-reinstall git+https://github.com/EpistasisLab/tpot.git@development
```

**Comment by weixuanfu** [comment]
The issue has been fixed in new version of TPOT (v0.11.0). Please feel free to reopen it if you have any questions or suggestions.
